task/tkn-bundle/0.2/migrations/0.2.1.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Created for task: tkn-bundle@0.2
+# Creation time: 2025-08-15
+
+declare -r pipeline_file=${1:?missing pipeline file}
+
+# Check if depth parameter already exists in task clone-repository if tkn-bundle* task exists in pipeline
+if yq -e '.spec.tasks[] | select(.name == "tkn-bundle")' "$pipeline_file" >/dev/null && ! yq -e '.spec.tasks[] | select(.name == "clone-repository").params[] | select(.name == "depth")' "$pipeline_file" >/dev/null; then
```

Comment:
The name in `.spec.tasks[]` is `build-container`.

Reply:
Good catch!